### ISSUE TITLE
paddle2.0-rc的LSTM网络问题，求大家帮忙看一下，非常感谢

### ISSUE THREAD

**Issue by wkzqn** [PaddlePaddle/Paddle#28543]
使用GPU报core dumped的错误，使用cpu可以正常运行:
```
import paddle
import paddle.fluid as fluid

use_gpu = True
place = fluid.CUDAPlace(0) if use_gpu else fluid.CPUPlace()
with fluid.dygraph.guard(place):
    rnn = paddle.nn.LSTM(100, 100)
    opt = fluid.optimizer.AdamOptimizer(learning_rate=0.001, parameter_list=rnn.parameters())
    x = paddle.randn((64, 10, 100))
    y, (h, c) = rnn(x)
    x = h[-1, :, :]  # x shape is: [64, 100]
    loss = -1 * fluid.layers.mean(x)
    loss.backward()
    opt.minimize(loss)
    rnn.clear_gradients()
    print("成功")
```

**Comment by wangxicoding** [comment]
GPU可以正常跑，估计是你的环境有些问题，把错误log发一下吧~~
![image](https://user-images.githubusercontent.com/10208305/98794577-2a3dc780-2444-11eb-83f1-5855bfb9fef3.png)

**Comment by wkzqn** [comment]
![image](https://user-images.githubusercontent.com/44158381/98801690-22365580-244d-11eb-871b-224a6996fc27.png)

**Comment by wkzqn** [comment]
我用linear层可以跑，用lstm不行。
```
import paddle
import paddle.fluid as fluid

use_gpu = True
place = fluid.CUDAPlace(0) if use_gpu else fluid.CPUPlace()
with fluid.dygraph.guard(place):
    rnn = paddle.nn.Linear(100, 100)
    opt = fluid.optimizer.AdamOptimizer(learning_rate=0.001, parameter_list=rnn.parameters())
    x = paddle.randn((64, 100))
    y = rnn(x)

    loss = -1 * fluid.layers.mean(x)
    loss.backward()
    opt.minimize(loss)
    rnn.clear_gradients()
    print("success!")
```
@wangxicoding

**Comment by wangxicoding** [comment]
什么版本的paddle，试下2.0rc版本

**Comment by wkzqn** [comment]
就是2.0-rc。我是用的百度的ai studio这个平台，不过会有这么一句警告，不知道有没有啥问题。
```
You are using Paddle compiled with TensorRT, but TensorRT dynamic library is not found. Ignore this if TensorRT is not needed.
```
@wangxicoding

**Comment by wkzqn** [comment]
![Uploading image.png…]()

**Comment by wkzqn** [comment]
![image](https://user-images.githubusercontent.com/44158381/98806115-a8ee3100-2453-11eb-8a36-0251c7538e98.png)

**Comment by wangxicoding** [comment]
我再aistudio上也复现了。。我找相关人员排查一下

**Comment by wkzqn** [comment]
@wangxicoding 谢谢你啦，感觉是一个bug，我最近正在利用ai studio做实验，希望快点修好。

**Comment by skywalk163** [comment]
建议用比较完整的代码来试试，我这边用你的代码也是cpu能运行，gpu直接就卡住（一直运行状态）
完整代码就是用类定义一个模型，至少下面的代码是能用的：
        import paddle
        vocab_size =100
        emb_size = 256
        seq_len = 200
        batch_size = 128
        epochs = 3

        input_size =emb_size
        hidden_size  = emb_size
        num_layers=3

        class MyNet(paddle.nn.Layer):
            def __init__(self):
                super(MyNet, self).__init__()
                self.emb = paddle.nn.Embedding(vocab_size, emb_size)
                self.fcx = paddle.nn.Linear(in_features=emb_size, out_features=2)
                self.rnn = paddle.nn.LSTM(input_size=input_size, hidden_size=hidden_size, num_layers=num_layers, dropout=0.3)
                
            def forward(self, x):
                x = self.emb(x)
                x ,(prev_h,prev_c)= self.rnn(x)
                x = paddle.mean(x, axis=1)

                x = self.fcx(x)
                return x
        model = paddle.Model(MyNet())

**Comment by wangxicoding** [comment]
@wkzqn 2.0rc1目前已经修复了
